.github/actions/create-jira-issues-for-community-activities/action.yml

```diff
@@ -100,21 +100,25 @@ runs:
       if: steps.issue-related-pr.outputs.is-issue-pr == 'False'
       shell: pwsh
       run: |
-        # Note that while --fail will fail the command if the request fails, it won't show the server output. Thus,
-        # remove for debugging. --fail-with-body provides that too, but is only available under curl >=7.76.0 which is
-        # not yet available for GitHub Actions, see https://github.com/Lombiq/GitHub-Actions/issues/53.
-        curl --request POST `
-          --url '${{ env.JIRA_BASE_URL }}/rest/api/3/issue/${{ steps.create-issue.outputs.issue }}/remotelink' `
-          --user '${{ env.JIRA_USER_EMAIL }}:${{ env.JIRA_API_TOKEN }}' `
-          --header 'Accept: application/json' `
-          --header 'Content-Type: application/json' `
-          --fail `
-          --data '{
-                \"object\": {
-                    \"url\":\"${{ steps.issue-details.outputs.link-url }}\",
-                    \"title\":\"${{ steps.issue-details.outputs.link-title }}\"
-                }
-            }'
+        $linkProperties = @{
+            object = @{
+                url = '${{ steps.issue-details.outputs.link-url }}'
+                title = '${{ steps.issue-details.outputs.link-title }}'
+            }
+        }
+        $requestParameters = @{
+            Uri = '${{ env.JIRA_BASE_URL }}/rest/api/3/issue/${{ steps.create-issue.outputs.issue }}/remotelink'
+            Method = 'POST'
+            ContentType = 'application/json'
+            Headers = @{
+                Accept = 'application/json'
+                Authorization = 'Basic ' + [Convert]::ToBase64String([Text.Encoding]::UTF8.GetBytes('${{ env.JIRA_USER_EMAIL }}:${{ env.JIRA_API_TOKEN }}'))
+                ContentType = 'application/json'
+            }
+            Body = ConvertTo-Json($linkProperties)
+        }
+
+        Invoke-WebRequest @requestParameters
```

Comment:
When this fails (e.g. the Jira user doesn't have access to the given project, or if the base URL is wrong), will it display the response output? If yes, then we can close https://github.com/Lombiq/GitHub-Actions/issues/53.

Reply:
Here are the responses:

Wrong email address or issue key doesn't exist:
```
Invoke-WebRequest: [...].ps1:20:13
Line |
  20 |  $response = Invoke-WebRequest @requestParameters
     |              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | {"errorMessages":["Issue does not exist or you do not have permission to see it."],"errors":{}}
```

Wrong access token (probably it thinks that if it's not a valid token, then it's a password?):
```
Invoke-WebRequest: [...].ps1:20:1
Line |
  20 |  Invoke-WebRequest @requestParameters
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Basic authentication with passwords is deprecated.  For more information, see: https://developer.atlassian.com/cloud/confluence/deprecation-notice-basic-auth/ 
```

Link title missing:
```
Invoke-WebRequest: [...].ps1:20:1
Line |
  20 |  Invoke-WebRequest @requestParameters
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | {"errorMessages":[],"errors":{"title":"'Link Text' is required."}}
```

Link URL missing:
```
Invoke-WebRequest: [...].ps1:20:1
Line |
  20 |  Invoke-WebRequest @requestParameters
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | {"errorMessages":[],"errors":{"url":"'URL' is required."}}
```

Success prints out the response details, same as other scripts, so everything's covered!

Reply:
Looks good, we don't need https://github.com/Lombiq/GitHub-Actions/issues/53, then.